src/pip/_internal/commands/uninstall.py

```diff
@@ -35,19 +39,22 @@ class UninstallCommand(Command, SessionCommandMixin):
     def add_options(self):
         # type: () -> None
         self.cmd_opts.add_option(
-            '-r', '--requirement',
-            dest='requirements',
-            action='append',
+            "-r",
+            "--requirement",
+            dest="requirements",
+            action="append",
             default=[],
-            metavar='file',
-            help='Uninstall all the packages listed in the given requirements '
-                 'file.  This option can be used multiple times.',
+            metavar="file",
+            help="Uninstall all the packages listed in the given requirements "
+            "file.  This option can be used multiple times.",
         )
         self.cmd_opts.add_option(
-            '-y', '--yes',
-            dest='yes',
-            action='store_true',
-            help="Don't ask for confirmation of uninstall deletions.")
+            "-y",
+            "--yes",
+            dest="yes",
+            action="store_true",
+            help="Don't ask for confirmation of uninstall deletions.",
+        )
```

Comment:
Please don't change formatting on the rest of the code -- it makes this PR exceedingly difficult to review.

See https://dev.to/mpermar/how-to-create-effective-pull-requests-2m8e#avoid-formatting-changes a longer-form explanation. :)